src/Model/Filter/Like.php

```diff
@@ -15,7 +15,9 @@ class Like extends Base
         'before' => false,
         'after' => false,
         'mode' => 'or',
-        'comparison' => 'LIKE'
+        'comparison' => 'LIKE',
+        'wildcardAny' => '*',
```

Comment:
If default value of `wildcardAny` is kept `*` it will require a major version bump for the plugin as it's a non BC change.